README.md

```diff
@@ -1,107 +1,64 @@
-| `lint:format`    | Formats all files except the file list of `.prettierignore`.                       |
-| `test`           | Runs testing.                                                                      |
-| `test:watch`     | Runs an interactive test watcher.                                                  |
-| `test:cov`       | Runs testing with code coverage reports.                                           |
-| `test:update`    | Updates jest snapshot.                                                             |
-
-## App Structure
-
-Here is the structure of the app, which serves as generally accepted guidelines and patterns for building scalable apps.
-
-```
-.
-├── public                        # Express server static path and Webpack bundles output
-│   ├── favicon.ico               # App favicon
-│   ├── logo192.png               # App logo small
-│   ├── logo512.png               # App logo large
-│   └── manifest.json             # App favicon and logo manifest
-├── src                           # App source code
-│   ├── config                    # App configuration by environments
-│   │   ├── default.ts            # Default settings
-│   │   ├── index.ts              # Configuration entry point
-│   │   └── prod.ts               # Production settings (overrides the default)
-│   ├── components                # Reusable components
-│   ├── pages                     # Page components
-│   ├── app                       # App root component
-│   ├── store                     # Redux store creator, actions + reducers (a.k.a slice)
-│   ├── services                  # API calls
-│   ├── utils                     # App-wide utils (e.g. mock store creator for testing etc.)
-│   ├── static                    # Static assets (e.g. images, fonts etc.)
-│   ├── theme                     # App-wide style and vendor CSS framework
-│   ├── types                     # App-wide type definitions
-│   ├── client                    # App bootstrap and rendering (Webpack entry)
-│   ├── routes                    # Routes configuration for both client-side and server-side
-│   └── server                    # Express server (with Webpack dev and hot middlewares)
-├── webpack                       # Webpack configurations
-├── jest                          # Jest configurations
-├── babel.config.js               # Babel configuration
-├── tsconfig.json                 # TypeScript configuration
-├── postcss.config.js             # PostCSS configuration
-├── .eslintrc.js                  # ESLint configuration
-├── .stylelintrc.js               # stylelint configuration
-└── nodemon.json                  # nodemon configuration
-```
+-   Commiting to the main branch is forbidden. Make use of topic branches to change code.
+-   PR names should follow the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) (`fix(#x)`, `feat(#y)`, `refactor(#z)`, …)
```

Comment:
for the commits I use a lot https://gitmoji.dev/ :) it's more funny and clear when  you look in the commit timeline. I think it's a great things to put the issue #x in the commits but in the PR name / Branch name I don't think it's a good practice. It's better to use the description of the PR ?

Reply:
I usually use the issue # on the branch name to have a quick look at it when commiting. It spares me looking it up on the website. Totally up to gitmoji!